php/utils.php

```diff
@@ -24,7 +24,7 @@ function extract_from_phar( $path ) {
 
 	$fname = basename( $path );
 
-	$tmp_path = get_temp_dir() . "wp-cli-$fname";
+	$tmp_path = get_temp_dir() . "wp-cli-$fname-" . uniqid( 'wp-cli-extract-from-phar-', true );
```

Comment:
I think it would be better to do this as
```
get_temp_dir() . uniqid( "wp-cli-extract-from-phar-", true ) . "-$fname";
```
to keep the extension the same.

Reply:
What about:

```
get_temp_dir() . 'wp-cli-' . uniqid( "wp-cli-extract-from-phar-", true ) . "-$fname";
```

Reply:
The `wp-cli-` bit though is covered by the arg to `uniqid()`

Reply:
Ah, I see:

```
local ➜  wordpress-develop git:(master) wp shell
wp> $fname = 'router.php';
=> string(10) "router.php"
wp> 'wp-cli-' . uniqid( "wp-cli-extract-from-phar-", true ) . "-$fname";
=> string(66) "wp-cli-wp-cli-extract-from-phar-5a8b0566008283.78459585-router.php"
```